pyproject.toml

```diff
@@ -47,6 +47,7 @@ ansys-units = "^0.3.3"
 docker = ">=7.1.0"
 grpcio = "^1.30.0"
 grpcio-health-checking = "^1.30.0"
+grpcio-status = "^1.30.0"
```

Comment:
Provides the compiled code for [status.proto](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) and [error_details.proto](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto). We will include those protos in Fluent.